proto/ardaos/compliance/query.proto

```
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package ardaos.compliance;
+
+import "amino/amino.proto";
```

Buf (1.55.1)

import "amino/amino.proto": file does not exist

(COMPILE)